packages/client/components/ActivityLibrary/ActivityDetailsSidebar.tsx

```diff
@@ -205,16 +207,20 @@ const ActivityDetailsSidebar = (props: Props) => {
         <div className='mb-6 text-xl font-semibold'>Settings</div>
 
         <div className='flex grow flex-col gap-2'>
-          <NewMeetingTeamPicker
-            positionOverride={MenuPosition.UPPER_LEFT}
-            onSelectTeam={(teamId) => {
-              const newTeam = availableTeams.find((team) => team.id === teamId)
-              newTeam && setSelectedTeam(newTeam)
-            }}
-            selectedTeamRef={selectedTeam}
-            teamsRef={availableTeams}
-            customPortal={teamScopePopover}
-          />
+          {category === 'oneonone' ? (
+            <>TODO: New user picker will be here</>
+          ) : (
```

Comment:
+1 Maybe continue using the normal team picker, with a comment TODO to replace it with the user picker?